ci/ovs_test_setup/ovs_host.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+kubectl delete ns ovs
```

Comment:
is cleanup always required if it's done in a GH action ? 
I'm fine to leave it in, just wondering if it throws an error everytime about no "ns" called ovs...

Reply:
this script can be also manually run,